gr-fft/lib/ctrlport_probe_psd_impl.cc

```diff
@@ -49,17 +49,17 @@ void ctrlport_probe_psd_impl::forecast(int noutput_items,
         ninput_items_required[i] = d_len;
 }
 
-//    boost::shared_mutex mutex_buffer;
-//    mutable boost::mutex mutex_notify;
-//    boost::condition_variable condition_buffer_ready;
+//    std::mutex mutex_buffer;
+//    mutable std::mutex mutex_notify;
+//    std::condition_variable condition_buffer_ready;
```

Comment:
Can this be removed?

Reply:
Yep. It's not that hard to consult the header file. :smile:

Removed.